scss/config/_abstracts.scss

```diff
@@ -69,6 +69,6 @@ $functions: (
     @include herman-add($type, $name, $value, get-function('ratio'));
   } @else if index('font' 'fonts', $type) {
     @include add-font($name, $value);
-    @include herman-add($type, $name, $value, get-function('font'));
+    @include herman-add($type, $name, $value);
```

Comment:
Ah, Herman was trying to compile each map-value as a font (like it would with colors). What we really need to do is compile the entire map as a single font before passing it along to Herman. This should be fixed in 77cb5b1 (with a few other small bugfixes)